Src/BootCamp.Chapter/PcFactory.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using BootCamp.Chapter.Computer;
+
+namespace BootCamp.Chapter
+{
+    public abstract class PcFactory
+    {
+        protected Body body;
+        protected Cpu cpu;
+        protected Gpu gpu;
+        protected HardDisk disk;
+        protected Motherboard board;
+        protected Ram ram;
+
+        public DesktopComputer Assemble()
+        {
+            PrepareBody();
```

Comment:
Attach (Assemble) a part to the pc.

Reply:
Right now it kinda does it but you need to either:
A) return a created part
B) pass a PC and install the created part in the passed empty PC

Reply:
I changed the methods and return the parts